notebooks/002-openvino-api/002-openvino-api.ipynb

```diff
@@ -19,6 +19,8 @@
     "- [Loading a Model](#Loading-a-Model)\n",
     "  - [OpenVINO IR Model](#OpenVINO-IR-Model)\n",
     "  - [ONNX Model](#ONNX-Model)\n",
+    "  - [PaddlePaddle Model](#PaddlePaddle-Model)\n",
+    "  - [TensorFlow Model](#TensorFlow-Model)\n",
     "- [Getting Information about a Model](#Getting-Information-about-a-Model)\n",
     "  - [Model Inputs](#Model-Inputs)\n",
```

Comment:
Any of these URLs doesn't work

Reply:
they works in jupyter lab only, github can not use them, unfortunately.
I sow another way to make local hyperlinks in notebooks, described here https://medium.illumidesk.com/jupyter-notebook-little-known-tricks-b0866a558017
If it works, so I think it is good idea to have interactive table of contents for all notebooks

Reply:
looks like it does not supported by github preview, here explanation: https://stackoverflow.com/questions/38132862/html-anchors-in-a-jupyter-notebook-on-github

But works in binder, collab, jupyter, vscode and openvino docs.
The solution provided on stackoverflow breaks work in other places with opening notebook copy using browser, so I think it is better to stay it as is

Reply:
so let's stay with it